site/content/docs/features/config-file.md

```diff
@@ -0,0 +1 @@
+### TODO
```

Comment:
We have several files in this state; they are placeholders for new or adapted content.
They have been included so that the doc structure is evident, and also so that various links from real content have real targets.